phy/cluster/manual/clustering.py

```diff
@@ -6,15 +6,14 @@
 # Imports
 #------------------------------------------------------------------------------
 
-from collections import namedtuple, defaultdict, OrderedDict
-from copy import deepcopy
-
 import numpy as np
 
-from ...ext.six import iterkeys, itervalues, iteritems
 from ...utils.array import _as_array, _is_array_like
```

Comment:
I don't have strong opinions. I just followed what the other packages I know do (vispy, numba, scikit-learn, etc). I don't think it's worth changing everything now.